.vscode/launch.json

```diff
@@ -17,7 +17,7 @@
         "CALVA_DEV_GA": "FUBAR-69796730-4",
         "DEBUG": "no-debug-universal-analytics"
       },
-      "outFiles": ["${workspaceFolder}/dist/**/*.js"]
+      "outFiles": ["${workspaceFolder}/out/**/*.js"]
```

Comment:
@PEZ I noticed that changeing `dist` to `out` here made breakpoints be hit again for me. I went ahead and changed this in the other launch configs too, where it was still `dist`, since we don't have a `dist` directory anyway.

Reply:
Ah, cool. I ran in to this problem too. Was about to make a separate PR, but now I don't have to. 😄 Thanks! 🙏